data/json/items/comestibles/carnivore.json

```diff
@@ -216,6 +253,28 @@
     }
   },
   {
+    "id": "jerky_insect",
+    "copy-from": "meat_insect",
+    "type": "COMESTIBLE",
+    "name": "insect jerky",
+    "name_plural": "insect jerky",
+    "description": "Salted, dried insect meat that never goes bad, but will make you thirsty.",
+    "weight": 40,
+    "container": "bag_plastic",
+    "color": "brown",
+    "charges": 3,
+    "stack_size": 3,
+    "parasites": 0,
+    "healthy": 0,
+    "spoils_in": 0,
```

Comment:
So you don't need to put in things as 0 that are already defaulted as such. Like "healthy", "spoils_in", or "parasites".